keps/sig-network/4444-service-routing-preference/README.md

```diff
@@ -339,34 +295,37 @@ NOTE: Implementations reserve the right to refine the behavior associated with
 
 ### Notes/Constraints/Caveats (Optional)
 
-<!--
-What are the caveats to the proposal?
-What are some important details that didn't come across above?
-Go in to as much detail as necessary here.
-This might be a good place to talk about core concepts and how they relate.
--->
+None.
```

Comment:
Caveat: This being our third attempt at an API for this, there's a non-zero chance that we STILL didn't get it right, and will have to revisit again.

Caveat: Converting from the annotation to the field is not quite a mechanical change, which means users will have to think about it.

Reply:
Added 1.
2 is part of Risks and Mitigations.